src/redis/mod.rs

```diff
@@ -1,47 +1,5 @@
-use testcontainers::{core::WaitFor, Image};
+mod default;
```

Comment:
let's use `standalone` instead of default? The module is private, so only for better understanding

Reply:
I wasn't happy with default either but couldn't come up with an alternative in the moment. `standalone` is a lot better.